pkg/search/parser.go

```diff
@@ -84,6 +84,50 @@ func GetFieldValueFromQuery(query string, label FieldLabel) (string, bool) {
 	return "", false
 }
 
+func splitQuery(query string) []string {
+	var pairs []string
+	var previousEnd, previousPlusIndex int
+	prevDoubleQuoteCount := 0
+	for i, rune := range query {
+		if rune == ':' && previousPlusIndex != 0 {
+			if previousEnd > previousPlusIndex {
+				continue
+			}
+
+			pairs = append(pairs, query[previousEnd:previousPlusIndex])
+			previousEnd = previousPlusIndex + 1
+			continue
+		}
+		if rune == '"' {
```

Comment:
We can just use a flag for inside double quotes right? None of our keywords should matter if we're inside a "

Reply:
Do you mean use a set a bool flag instead of the doubleQuotesCount int ?